test/integration/echo_integration_test.cc

```diff
@@ -3,19 +3,36 @@
 #include "test/server/utility.h"
 
 namespace Envoy {
+
+std::string echo_config;
+
 class EchoIntegrationTest : public BaseIntegrationTest,
                             public testing::TestWithParam<Network::Address::IpVersion> {
 public:
-  EchoIntegrationTest() : BaseIntegrationTest(GetParam()) {}
+  EchoIntegrationTest() : BaseIntegrationTest(GetParam(), echo_config) {}
+
+  // Called once by the gtest framework before any EchoIntegrationTests are run.
+  static void SetUpTestCase() {
+    echo_config = ConfigHelper::BASE_CONFIG + R"EOF(
```

Comment:
nit: Could this just be a const member variable?

Reply:
I don't think so - when I tried it that way it segfaulted as it created the local echo_config before BASE_CONFIG was initialized.  Could have been PEBAC of some sort - I can give it another shot if you think it ought to work

Reply:
It seems like it should work but it's not a big deal. I wouldn't worry about it.